spec/lib/client_spec.rb

```diff
@@ -319,7 +319,7 @@
 
   describe "#update" do
     before(:each) do
-      @body = "<tns:update><tns:sObjects><ins0:type>Opportunity</ins0:type><ins0:Id>003ABCDE</ins0:Id><tns:Name>SOAPForce Opportunity</tns:Name><tns:CloseDate>2013-08-12</tns:CloseDate><tns:StageName>Closed Won</tns:StageName></tns:sObjects></tns:update>"
+      @body = "<tns:update><tns:sObjects><ins0:type>Opportunity</ins0:type><tns:Id>003ABCDE</tns:Id><tns:Name>SOAPForce Opportunity</tns:Name><tns:CloseDate>2013-08-12</tns:CloseDate><tns:StageName>Closed Won</tns:StageName></tns:sObjects></tns:update>"
```

Comment:
I didn't expect the namespace to change. Any idea why it did?

Reply:
No, not really, but it looks like a Savon issue https://github.com/savonrb/savon/issues/830